recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
     - setuptools
   run:
     - python
-    - numpy
+    - numpy x.x
```

Comment:
> Is numpy only pinned if we build against it?

Yep. See http://conda-forge.github.io/docs/meta.html#building-against-numpy

`defaults` has a few bogus numpy build strings that caused problems in the past. We should use them only when really needed to avoid the burden on the CIs and unnecessary downloads when using the package. Not to mention that we would need to update the matrix every new numpy version with really needing it.